examples/siro_sandbox/controllers.py

```diff
@@ -194,14 +194,8 @@ def act(self, obs, env):
         KeyNS = GuiInput.KeyNS
         gui_input = self._gui_input
 
-        should_end = False
-        should_reset = False
-
-        if gui_input.get_key_down(KeyNS.ESC):
-            should_end = True
-        elif gui_input.get_key_down(KeyNS.M):
-            should_reset = True
-        elif gui_input.get_key_down(KeyNS.N):
+        # todo: remove this or fix navmesh visualization (doesn't work with replay-rendering)
```

Comment:
It doesn't work with replay-rendering but nav mesh is visible on the 3-rd person debug images view. It might be useful for debugging to keep this.

Reply:
Sounds good. I'll update the comment.